tests/dailytest/case.go

```diff
@@ -351,34 +351,40 @@ func updatePKUK(db *sql.DB, opNum int) error {
 	maxKey := 20
 	mustExec(db, "create table pkuk(pk int primary key, uk int, v int, unique key uk(uk));")
 
-	var pks []int
+	pks := make(map[int]struct{})
+	freePks := make([]int, maxKey)
```

Comment:
How about `freePks := rand.Perm(maxKey)`?

Reply:
Cool, I didn't know about that, thanks.